e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/QuestionsAndAnswersE2eTest.java

```diff
@@ -82,7 +82,7 @@ void shouldReturnQuestionId()
                     "question": "What is the meaning of life?",
                     "agentId": "agent-id",
                     "restrictionQuery": {
-                        "nodesIds": ["node-id"]
+                        "nodesIds": ["1a0b110f-1e09-4ca2-b367-fe25e4964a4e"]
```

Comment:
I guess this is a real noderef from the sample site? If so then please could you extract it to a named constant so we know what it is?

An alternative would be to create a node for use in this test.

Reply:
yep, its a real noderef from sample site, I extracted it to the constant